tools/build_nbr.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+#echo "**************** set build_nbr"
+COMMIT=`git log -1 | grep -x "commit .*" | cut -c 8-14`
+CDATE=`git log -1 | grep -x "Date: .*" | cut -c 9-38`
+
+cat << EOF > src/commit.ml
+(* $Id: commit.ml,v 7.00 2017-06-05 07:59:58 hg Exp $ *)
+(* Copyright (c) 1998-2017 INRIA *)
```

Comment:
Though, not I care very much, but this file was created by you, so the copyright is not correct.